webpages/settings/index.html

```diff
@@ -31,287 +104,22 @@
-              <button
-                type="preset-button"
-                class="large-button"
-                :disabled="!addon._enabled"
-                @click="loadPreset(preset)"
-                :title="preset.description"
-              >
-                {{preset.name}}
-              </button>
-            </div>
-          </div>
-        </div>
-      </div>
-    </template>
-    <template id="addon-tag-component">
-      <div
-        class="badge"
-        v-if="shouldShow"
-        :class="{
-        'tooltip': tagInfo.tooltipText,
-        'blue': tagInfo.color === 'blue',
-        'yellow': tagInfo.color === 'yellow',
-        'red': tagInfo.color === 'red',
-        'darkred': tagInfo.color === 'darkred',
-        'green': tagInfo.color === 'green',
-        'darkgreen': tagInfo.color === 'darkgreen',
-        'purple': tagInfo.color === 'purple'
-        }"
-      >
-        {{ tagName }}
-        <span v-if="tagInfo.tooltipText" class="tooltiptext">{{ tagTooltip }}</span>
-      </div>
-    </template>
-    <template id="addon-group-component">
-      <div class="addon-group" v-show="shouldShow">
-        <span @click="toggle">
-          <img src="../../images/icons/expand.svg" alt="v" :class="{ 'reverted': group.expanded }" />
-          {{ group.name }} ({{ shownCount }})</span
-        >
-        <addon-body
-          v-for="addonId of group.addonIds"
-          :addon="manifestsById[addonId]"
-          :group-id="group.id"
-          :group-expanded="group.expanded"
-        ></addon-body>
-      </div>
-    </template>
-    <template id="category-selector-component">
-      <div
-        class="category"
-        :class="{
-          sel: category.id === selectedCategory,
-          hasParent: category.parent,
-        }"
-        v-show="shouldShow"
-        transition="expand"
-        :style="{ marginBottom: category.marginBottom ? '12px' : 0 }"
-        @click="onClick($event)"
-      >
-        <img :src="'../../images/icons/' + category.icon + '.svg'" />
-        <span>{{ category.name }}</span>
-      </div>
-    </template>
+    <script src="../../libraries/thirdparty/color-picker.js" type="module"></script>
     <script src="index.js" type="module"></script>
   </head>
-  <body style="display: none" v-show="loaded" :class="{light: theme}">
+  <body style="display: none" v-show="isIframe && loaded || !isIframe" :class="{light: theme}">
     <div class="navbar">
-      <img :src="switchPath" class="toggle" @click="sidebarToggle()" v-show="smallMode" alt="Logo" />
+      <img
+        style="display: none"
+        :src="switchPath"
+        class="toggle"
+        @click="sidebarToggle()"
+        v-show="smallMode"
+        alt="Logo"
+      />
       <img src="../../images/icon.svg" class="logo" alt="Logo" />
-      <h1>{{ msg("settings") }}</h1>
-      <img @click="setTheme(!theme)" class="theme-switch" v-bind:src="themePath" />
+      <h1 style="display: none" v-show="true">{{ msg("settings") }}</h1>
```

Comment:
v-cloak, people

Reply:
I’m dumb but what does v-cloak do

Reply:
Vue automatically removes v-cloak when it loads. If you put a css rule to hide v-cloak items, you don't need v-show or display:none multiple times.

Reply:
@GrahamSH-LLK But then the elements are visible until Vue loads

Reply:
Could be fixed with
```css
[v-cloak] {
  display: none;
}
```

Reply:
Doesn't the body have display none?

Reply:
> @GrahamSH-LLK But then the elements are visible until Vue loads

> If you put a css rule to hide v-cloak items